ctapipe/image/tests/test_time_next_neighbor_cleaning.py

```diff
@@ -34,3 +34,17 @@ def test_clean(example_subarray, example_event):
         camera.geometry, image, peaktime, sample_time, sum_time=1 * u.ns
     )
     assert len(mask) == camera.geometry.n_pixels
+
+
+def test_calculate_ipr(tmp_path):
+    source = EventSource("dataset://gamma_test_large.simtel.gz")
```

Comment:
You can use the file `lst_prod3_calibration_and_mcphotons.simtel.zst` with the option `skip_calibration_events=False` to actually use pedestal events for this test.